target/linux/ath79/dts/ar9344_atheros_db120.dts

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
```

Comment:
dts-v1 has been moved to ath79.dtsi, so it can be dropped here.

But you have to move the ar9344.dtsi include before the others ...